### PR TITLE
Complete first homework

### PR DESCRIPTION
**Pull request by TheRockShade**

### CODE REVIEW

File: ui/homework/homework_1/test1.ts [hunk 1]
```diff
@@ -4,30 +4,33 @@
 */
 
 type Person = {
-  occupation?: string;
+    name: string;
+    age: number | 'unknown';
```
Comment (aleksandrashmoylova) [inline ui/homework/homework_1/test1.ts:8]: are you sure about this type?

Comment (TheRockShade) [reply ui/homework/homework_1/test1.ts:8]: You mean "occupation" or "age"?

Comment (aleksandrashmoylova) [reply ui/homework/homework_1/test1.ts:8]: no, I mentioned age here

Comment (TheRockShade) [reply ui/homework/homework_1/test1.ts]: Hope it's right now

**Review by aleksandrashmoylova — APPROVED** [review]

**Pull request merged**